planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -8,6 +8,25 @@
       "explain format = 'brief' select * from t where b > 'a' order by b limit 2"
     ]
   },
+  {
+    "name": "TestVerboseExplain",
+    "cases": [
```

Comment:
Please add more test cases about `IndexLookUp` and `MergeJoin`.

Reply:
Ok